test/libsolidity/semanticTests/externalContracts/strings.sol

```diff
@@ -0,0 +1,76 @@
+==== ExternalSource: _stringutils/stringutils.sol ====
+==== Source: strings.sol ====
+pragma abicoder v2;
+import "_stringutils/stringutils.sol";
+
+contract test {
```

Comment:
Is this test meant to cover the whole library? I see that for example `nextRune()` is not tested.

Reply:
The goal isn't to provide a comprehensive test suite for random external sources, but to create some (semi-)real life use case examples and measure the gas usage of them with codegen v1 and v2.

Reply:
Fair enough.